MonoGame.Framework/Graphics/PackedVector/Short2.cs

```diff
@@ -115,8 +115,8 @@ private static uint PackInTwo (float vectorX, float vectorY)
 			const float minNeg = ~(int)maxPos; // two's complement
 
             // clamp the value between min and max values
-            var word2 = ((uint) MathF.Round(MathHelper.Clamp(vectorX, minNeg, maxPos)) & 0xFFFF);
-            var word1 = (((uint) MathF.Round(MathHelper.Clamp(vectorY, minNeg, maxPos)) & 0xFFFF) << 0x10);
+            var word2 = (uint)((int) MathF.Round(MathHelper.Clamp(vectorX, minNeg, maxPos)) & 0xFFFF);
+            var word1 = (uint)(((int) MathF.Round(MathHelper.Clamp(vectorY, minNeg, maxPos)) & 0xFFFF) << 0x10);
```

Comment:
I guess this was also broken code spotted by the unit test runs?

Reply:
Yes it was failing one of the unit tests.